### ISSUE TITLE
extraOutputs throws error if unused in handler (v4)

### ISSUE THREAD

**Issue by twain09** [Azure/azure-functions-nodejs-library#71]
When using `extraOutputs` on an httpTrigger i get an error message if this output is not used inside the handler.

For example the following code throws:
**Error message testing locally**
`[2023-03-31T07:21:24.197Z] Executed 'Functions.ping' (Failed, Id=c1bd6ece-961f-44df-86cb-34ebe0310847, Duration=155ms)`
`[2023-03-31T07:21:24.200Z] System.Private.CoreLib: Exception while executing function: Functions.ping. Microsoft.Azure.WebJobs.Script.Grpc: Unknown ParameterBindingType.`

**Error message when deployed**
`2023-03-31T07:37:35Z   [Error]   Executed 'Functions.ping' (Failed, Id=d28c8178-0784-4f89-b234-ee23544da68d, Duration=109ms)`

**Code**
```
const cosmosOutput = output.cosmosDB({
    connectionStringSetting: 'CosmosDbConnectionString',
    databaseName: 'test',
    collectionName: 'test'
});

app.http('ping', {
    methods: ['GET'],
    authLevel: 'function',
    extraOutputs: [cosmosOutput],
    handler: (request, context): Promise<HttpResponseInit> => {
        return;
    }
});
```

**Working example**
If i do the same thing but using the output at least once everything works fine. So this would work for example
```
const cosmosOutput = output.cosmosDB({
    connectionStringSetting: 'CosmosDbConnectionString',
    databaseName: 'test',
    collectionName: 'test'
});

app.http('ping', {
    methods: ['GET'],
    authLevel: 'function',
    extraOutputs: [cosmosOutput],
    handler: (request, context): Promise<HttpResponseInit> => {
        context.extraOutputs.set(cosmosOutput, []);       // this line is needed
        return;
    }
});
```

Since i want the output binding only to be used in a certain `if statement`, it would be nice if i dont have to set a placeholder like mentioned in the working example.
I've only tested it with the httpTrigger, so i'm not sure if other triggers might as well have this problem.
I am also not 100% sure if this a bug in the new v4 model or if i doing something wrong?

**Comment by ejizba** [comment]
Hi @twain09 yes I believe this is a bug, thanks for reporting! We'll look into a fix

**Comment by ejizba** [comment]
FYI @twain09 we just released v4.0.0-alpha.10 of the npm package with this fix
